src/StaticAnalysis.ruleset

```diff
@@ -1,4 +1,14 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="Console Rules" Description="These rules enforce static analysis on console code." ToolsVersion="15.0">
+
   <Include Path="cppcorecheckrules.ruleset" Action="Default" />
+
+  <Rules AnalyzerId="Microsoft.Analyzers.NativeCodeAnalysis" RuleNamespace="Microsoft.Rules.Native">
+    <Rule Id="C6001" Action="Error" />
+    <Rule Id="C6011" Action="Error" />
+    <Rule Id="C6029" Action="Error" />
+    <Rule Id="C6053" Action="Error" />
+    <Rule Id="C26436" Action="Error" />
+  </Rules>

```

Comment:
Can we add C26495 here?

Reply:
(This is a first pass at making sure we can stand it up in CI and have it not take up too much time; do not consider the ruleset normative at the moment!)

Reply:
Got it.  Thanks!